scripts/prepare_automatic_backups.py

```diff
@@ -80,21 +95,33 @@ def update_cron_dict(cron_dict):
 
 
 def get_cron_dict():
+    """Converting yaml file to dict.
+
+    Returns:
+        dict(str, str). Returns yaml file in dict format.
```

Comment:
Please explain your reasoning for how you know that this is a dict(str, str). What guarantees that the values are always strings?